tech.keyforge.master.metainfo.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>tech.keyforge.master</id>
+  <developer id="tld.vendor">
```

Comment:
this is an INVALID developer ID

Reply:
developer ID removed

Reply:
Removing it was NOT the request.

Reply:
Ok, I get it. Now it should be fine. Thanks